### ISSUE TITLE
Instant 70 Plus bugged my armor ability

### ISSUE THREAD

**Issue by Healium7** [dalaranwow/dalaran-wow#4254]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Boosted my Paladin with Instant 70 Plus but chose mail because of the stats. Now I can only wear mail. No plate!

**Expected behaviour**: Tell us what should happen instead
Should be able to wear any armor as a Lvl 70 Paladin.

**Steps to reproduce the problem**:

1. Persistent even with relog!
[armor](https://cloud.githubusercontent.com/assets/25143644/21996171/884ccf08-dbee-11e6-8a0e-6ce620616860.jpg)

2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Edzero** [comment]
You have to learn it from the class trainer.
